configs/webserver/sample.conf

```diff
@@ -54,6 +54,8 @@ mask_user_info = false
 enable_2FA = false
 # Force enable 2-Factor-Authentication (TOTP).
 force_2FA = false
+# This image is used to launch ssh session (using system role image) to support fast uploading.
```

Comment:
Please change the comment to:
Container image to use SSH/SFTP file upload when storage proxy configuration supports independent resource group for file transfer.